pac4j-saml/src/main/java/org/pac4j/saml/metadata/SAML2IdentityProviderMetadataResolver.java

```diff
@@ -71,27 +75,31 @@ public final MetadataResolver resolve() {
                     path = "/" + path;
                 }
                 resource = ResourceHelper.of(new ClassPathResource(path));
-            } else if (this.idpMetadataPath.startsWith("file:")) {
-                resource = ResourceHelper.of(new FileSystemResource(this.idpMetadataPath));
-            } else if (this.idpMetadataPath.startsWith("http")) {
+            }  else if (this.idpMetadataPath.startsWith("https")) {
```

Comment:
Good point. I think we might abstract out the loading of resources, to a different component. So that by default, there is a MetadataResourceLoader that does what we have today, and we can then pass in other impls of that component that allow for local dev/testing? perhaps a mocked one?